python/tvm/relay/frontend/onnx.py

```diff
@@ -1443,6 +1443,16 @@ def _impl_v11(cls, inputs, attr, params):
         return _op.image.resize(inputs[0], out_size, layout, method, coord_trans)
 
 
+class NonZero(OnnxOpConverter):
+    """Operator converter for NonZero
+    """
+    @classmethod
+    def _impl_v9(cls, inputs, attr, params):
+        if len(inputs) > 1:
+            raise ValueError("Expect 1 input only")
+        return AttrCvt(op_name='argwhere')(inputs, attr, params)
```

Comment:
The output shape of argwhere is different from the one of ONNX nonzero.  Needs transpose here.

Reply:
Thanks for the review and I updated it.